opencga-app/app/scripts/docker/opencga-daemon/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM opencga
 
 VOLUME /opt/opencga/conf
 VOLUME /opt/opencga/sessions
+VOLUME /opt/opencga/variants
 
 USER 1001:1001
 
-CMD [". /opt/opencga/conf/opencga-env.sh && echo ${OPENCGA_PASS} | /opt/opencga/bin/opencga-admin.sh catalog daemon --start"]
+CMD ["-c" ,"echo ${OPENCGA_PASS} | /opt/opencga/bin/opencga-admin.sh catalog daemon --start"]
```

Comment:
Did you decide you preferred this rather than overriding the ENTRYPOINT?

Reply:
Yes, either works? thoughts?

Reply:
Reason was password is less visible.

Reply:
ti's fine just wondered if a reason arose